packages/lib/delivery/src/Notification.ts

```diff
@@ -90,3 +93,98 @@ const getOtpContentForNotificationChannel = (
             );
     }
 };
+
+// method to resend otp
+export const resendOtp = async (
```

Comment:
why is it called resend and not send otp ?

Reply:
Yes, can be send OTP that's more better